### PR TITLE
Fix photo credit link

### PR DESCRIPTION
**Pull request by ggolubic**
#6 
Hey,

Fixed the link and also added target="_blank" so it opens in a new tab and rel="noopener" for security reasons.

### CODE REVIEW

File: src/templates/blog-post.js [hunk 1]
```diff
@@ -43,7 +43,9 @@ const PostFeaturedImage = ({ post }) => {
   let featuredImageByline
   if (featuredImageCredit) {
     if (featuredImageLink) {
-      featuredImageByline = <a href="{featuredImageLink}">{featuredImageCredit}</a>
+       <a href={featuredImageLink} target="_blank" rel="noopener">
```
Comment (merit-mthompson) [inline src/templates/blog-post.js:46]: We want to keep our variable assignment here, so we can use it below.

**Review by merit-mthompson — CHANGES_REQUESTED** [review]

**Comment by merit-mthompson** [comment]
@ggolubic thanks for the PR! This was addressed in PR #7 and we've merged/pushed that. If you'd like to update this PR to use the changes from that along with your target/rel changes we'd be happy to accept the PR!

File: src/templates/blog-post.js (hunk 1 above)
Comment (ggolubic) [reply src/templates/blog-post.js]: My bad, not sure how I missed that.

**Review by merit-mthompson — APPROVED** [review]

**Comment by merit-mthompson** [comment]
Looks good, thank you! Merging now.

**Pull request merged**